pkg/executables/kubectl.go

```diff
@@ -1323,6 +1324,15 @@ func (k *Kubectl) getObject(ctx context.Context, resourceType, name, namespace,
 	return nil
 }
 
+func (k *Kubectl) GetEksdRelease(ctx context.Context, name, namespace, kubeconfigFile string) (*eksdv1alpha1.Release, error) {
+	obj := &eksdv1alpha1.Release{}
+	if err := k.getObject(ctx, "release", name, namespace, kubeconfigFile, obj); err != nil {
```

Comment:
nit: maybe we should specify the group as well with "release" so it know how to distinguish from eks-a and eks-d releases